doc/book/src/patterns/id-pointer.md

```diff
@@ -0,0 +1,19 @@
+# ID Pointer
+
+ID Pointer is a technique which allows separation of the main data (an object) and its accessors / capabilities by linking the latter to the original. There's a few different directions in which this pattern can be used:
+
+- issuing transferable capabilities for shared objects (for example, a TransferCap that changes 'owner' field of a shared object)
+- splitting dynamic data and static (eg NFT and its Collection information)
```

Comment:
dynamic data and static data?
```suggestion
- splitting dynamic data and static (for example, an NFT and its Collection information)
```
why is Collection upper case? The way this is worded the NFT is the dynamic data?

Reply:
Uppercase because it's a Type.